backend/routes/pixel.go

```diff
@@ -65,18 +65,15 @@ func placePixelDevnet(w http.ResponseWriter, r *http.Request) {
     panic(err)
   }
 
-  x, err := strconv.Atoi(jsonBody["x"])
-  if err != nil {
-    panic(err)
-  }
-  y, err := strconv.Atoi(jsonBody["y"])
+  position, err := strconv.Atoi(jsonBody["position"])
   if err != nil {
     panic(err)
   }
+
   shellCmd := backend.ArtPeaceBackend.BackendConfig.Scripts.PlacePixelDevnet
-  position := x + y * int(backend.ArtPeaceBackend.CanvasConfig.Canvas.Width)
+  computedPosition := x + y * int(backend.ArtPeaceBackend.CanvasConfig.Canvas.Width)
```

Comment:
Since x & y are no longer defined, I think you just want to use the `position` value below

Reply:
Yeah true that was an oversight.
Will revert

Reply:
Since i will be passing the whole position from the frontend: x+y * width
Is this line still needed?
[ * int(backend.ArtPeaceBackend.CanvasConfig.Canvas.Width) ]

Reply:
Yeah I think this line can be removed, position will just be read from the jsonBody

Reply:
Okay, just pushed a new change kindly review.